Makefile

```diff
@@ -31,9 +31,7 @@ MAKEDIR:=$(strip $(shell dirname "$(realpath $(lastword $(MAKEFILE_LIST)))"))
 # Keep in sync with upup/models/cloudup/resources/addons/dns-controller/
 DNS_CONTROLLER_TAG=1.4.1
 
-ifndef VERSION
-  VERSION := git-$(shell git describe --always)
-endif
+VERSION?=dev
```

Comment:
So I like to run one version of nodeup and another version of kops.  This is because I use a build server to upload files to s3.  Will this still allow this?  I assume I can override.